source/agora/consensus/EnrollmentManager.d

```diff
@@ -586,7 +586,7 @@ unittest
 
         auto signature = gen_key_pair.secret.sign(hashFull(tx)[]);
         tx.inputs[0].signature = signature;
-        storage[hashFull(tx)] = tx;
+        storage[hashMulti(hashFull(tx), idx)] = tx;
```

Comment:
Another example why https://github.com/bpfkorea/agora/issues/501 would be great :)

Reply:
https://github.com/bpfkorea/agora/pull/523